addons/mail/static/src/models/suggested_partners/suggested_partners.js

```diff
@@ -0,0 +1,40 @@
+odoo.define('mail/static/src/models/suggested_partners/suggested_partners.js', function (require) {
+    'use strict';
+
+    const { registerNewModel } = require('mail/static/src/model/model_core.js');
+    const { attr, many2many, many2one } = require('mail/static/src/model/model_field.js');
+    const mailUtils = require('mail.utils');
+
+    function factory(dependencies) {
+        class SuggestedPartner extends dependencies['mail.model'] {
+            init(...args) {
+                super.init(...args);
+            }
+
+            static convertData(data) {
+                const parsedEmail = data[1] && mailUtils.parseEmail(data[1]);
+                const data2 = {
+                    checked: true,
+                    id: data[0],
+                    full_name: data[1],
+                    name: parsedEmail[0],
+                    email_address: parsedEmail[1],
+                    reason: data[2],
+                };
+                return data2;
+            }
+        }
+        SuggestedPartner.fields = {
+            id: attr(),
+            checked: attr({ default: true }),
+            full_name: attr(),
+            name: attr(),
+            email_address: attr(),
+            reason: attr(),
+        };
+        SuggestedPartner.modelName = 'mail.suggestedPartner';
+        return SuggestedPartner;
```

Comment:
I feel this model existence come having some data in the context of toggle-able suggestions in composer. Other than that. it should have related field to `mail.partner`.

Also convention for model name is camel_case, so `mail.suggested_partner`.